rfd/0002-code-of-conduct.md

```diff
@@ -0,0 +1,266 @@
+---
+authors: JP Bruins Slot <jpbruinsslot@gmail.com>
+state: discussion
+discussion: https://github.com/minvws/nl-kat-coordination/pull/3425
+labels: process
+---
+
+# RFD 0002: Code of Conduct: Code Review
+
+## Introduction
+
+Code reviews are an essential part of the software development process,
+ensuring that the codebase remains clean, efficient, and maintainable while
+fostering collaboration and knowledge sharing among team members. For both
```

Comment:
I think another major reason to do code reviews is to spot problems /  potential bugs in the new or changed code.